README.md

```diff
@@ -72,9 +72,11 @@ Listing of useful (mostly) public learning resources for machine learning applic
 
 ##### Upcoming:
 
-- [4th Machine Learning in High Energy Physics Summer School 2018](https://indico.cern.ch/event/687473/) (August 6-12, 2018)
+- [2nd PIP Summer School on Machine Learning 2018](http://www.pip.uni-bremen.de/news/pip-summer-school-2018/) (September 24-28, 2018)
```

Comment:
@SwapneelM Do you know more information about the [PIP program](http://www.pip.uni-bremen.de/news/pip-summer-school-2018/)? The website isn't very detailed in its description, but it doesn't seem to be focused on HEP. That's fine of course, but we probably want to focus the schools that we advertise to ones that are HEP focused and make their resources publicly available. (Along those lines I need to revise some of the deep learning ones below sometime).

Reply:
@matthewfeickert yeah, I did check it's not specifically for physics before adding it but I assumed since it is focused on the "physical sciences", that would be broad enough to cover it. Makes sense if we don't want to add those, though.

I do see DLSS/RLSS and older MLSS videos uploaded on videolectures.net (I'm at MLSS Madrid so I can confirm they're recording these, but the timeline for upload is unclear). Slides are available for PAISS and MLSS though.

I'll make a new commit removing the PIP school for now.

Reply:
PIP school removed in latest commit.